lib/hostel/site.rb

```diff
@@ -15,7 +15,9 @@ def domain
     end
     
     # Constructs a url for a path on this site
-    def build_path(path, secure: true, path_args: {})
+    # Note:  request_host is ignored now, but it is here for backward
+    # compatibility with old callers.
```

Comment:
Here is an example.  I am not sure I can find ALL the places that are now doing this, so for now, I think this is the best solution.

https://github.com/Americastestkitchen/treasure/blob/master/app/views/marketing_displays/element_templates/_user_details_condensed.html.haml#L47